hr_expense_invoice/wizard/expense_create_invoice.py

```diff
@@ -62,5 +62,5 @@ def create_invoice(self):
         invoice = (
             self.env["account.move"].with_context(type="purchase").create(invoice_vals)
         )
-        self.expense_ids.write({"invoice_id": invoice.id})
+        self.expense_ids.write({"invoice_id": invoice.id, "tax_ids": False})
```

Comment:
What about the final amount? Maybe it has been changed.

Reply:
Is not possible because total of sheet is computed according to lines (expenses).

Reply:
Not total of sheet, but the line.

I see this very weird... Are you saying that the invoice is assigned to all the lines? That's a mistake IMO.

Reply:
This is done through a wizard when the lines have been selected, so it is possible to assign the same invoice generated to several lines and then, the total cannot be assigned correctly.

Reply:
At least a constraint should be added to raise the error if the total amount doesn't match. If not, there will be problems. @kittiu what is your use case for adding such weird wizard?

Reply:
With my blurry memory. In v8, we are used to a module that create invoice from expense.
But to be frank, nowadays we didn't use it either. Please feel free to remove. It is erroneous to have too many ways to do the same.